### PR TITLE
update security configuration for environments

### PR DESCRIPTION
**Pull request by RuaidhriMcDonnell**
updated to correct config endpoints, and remove unnecessary additional settings

### CODE REVIEW

File: src/content/src/WebAPIService/appsettings.json [hunk 1]
```diff
@@ -10,9 +10,8 @@
     }
   },
   "ServiceConfigurationOptions": {
-    "Authority": "TBA",
+    "Authority": "https://security-sts.ci.eshopworld.net",
```
Comment (arturek) [inline src/content/src/WebAPIService/appsettings.json]: Will this value be ever used? It's overwritten everywhere.

Comment (RuaidhriMcDonnell) [reply src/content/src/WebAPIService/appsettings.json]: local machine...

Comment (arturek) [reply src/content/src/WebAPIService/appsettings.json]: Doesn't local machine use the Development profile?

**Comment by RuaidhriMcDonnell** [comment]
Gents, not sure what ye decided on your policy of squash / not squash, so i'll leave the merge to you

**Comment by dguerin** [comment]
@RuaidhriMcDonnell one final questions, if others have to fill in the details for the auth eg: scopes and such, why would we remove them from the config? 

It's also helpful as is for a guide.

**Comment by dguerin** [comment]
> > @RuaidhriMcDonnell one final questions, if others have to fill in the details for the auth eg: scopes and such, why would we remove them from the config?
> > It's also helpful as is for a guide.
> 
> I didn't. If you review the base settings file (`appsettings.json`) you'll see that those fields are still present. I removed them from the environment level files, as they are unnecessary there, as its highly unlikely that an app will define different scopes and API names for different environments, and the content in those files is clutter / unnecessary.

It does happen when teams mock endpoints in different environments and then use the "correct" endpoint in upper environments which is protected by different auth.

Having said that if they are in appsettings.json this change is fine to support the more common case.

**Pull request merged**